tests/forms_tests/tests/test_forms.py

```diff
@@ -2887,6 +2887,36 @@ class DateAgeForm(Form):
         form = DateAgeForm(data, initial={"date_age": ["200-10-10", 14]})
         self.assertTrue(form.has_changed())
 
+    def test_clean_disabled_multivalue(self):
+        """
+        #28882 -- Fixed bug with saving disabled MultiValueField
```

Comment:
Docstrings should state the expected behavior (although I'm not sure it's needed here as the behavior seems simple enough). Ticket references should be reserved for obscure issues (not needed here). These guidelines are from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).